notebooks/lesson03_data-cleaning.ipynb

```diff
@@ -0,0 +1,1866 @@
+{
```

Comment:
<p>isn't that the solution for the exercise?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lewtun/dslectures/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='67'/>